assignments/views.py

```diff
@@ -19,3 +21,18 @@ def get_queryset(self):
 class SectionsViewSet(NestedViewSetMixin, ReadOnlyModelViewSet):
     serializer_class = SectionSerializer
     queryset = Section.objects.filter(assignment__status=Assignment.STATUS_OPEN)
+
+
+class SubmitAnswersViewSet(ViewSet):
+
+    def get_serializer_context(self):
```

Comment:
Note that you are overriding an existing method here without calling super(). I would recommend that you do something like this to keep things consistent:

```
def get_serializer_context(self):
    context = super().get_serializer_context()
    context['assignment_slug'] = self.kwargs['slug']
    return context
```

assuming you are using Python 3, otherwise the super() call will look a bit different but still the same principle.